src/components/util/CodelabCard/styles.js

```diff
@@ -100,7 +100,7 @@ const useStyles = makeStyles((theme) => ({
     borderRadius: "500px",
   },
   logoImg: {
```

Comment:
Can you verify that there are no other element existed which uses this style class and this change will not break any other styles please?

Reply:
@shehand yes, I had already checked this before changing the CSS.

Reply:
@shehand can you please approve if all things are ready to go?